app/models/speaker_email_template.rb

```diff
@@ -0,0 +1,5 @@
+class SpeakerEmailTemplate
```

Comment:
Oh oops, well I just read the card a little more carefully, and it looks like Marty wants the template stuff out of the event model and into its own model, is that right?